helm/cluster-azure/templates/list.yaml

```diff
@@ -9,8 +9,6 @@
 ---
 {{- include "control-plane" . }}
 ---
-{{- include "machine-pools" . }}
```

Comment:
Ultimately it's your call. The reason I made this PR was that I wanted to simplify the schema as far as possible, and remove unused parts. You can always re-add the machine pool templating once it's needed again. (I learned at Giant Swarm that we usually don't want to carry unused code around)

Reply:
I am not sure if this is unused code since _we are supposed to move back to machinepools_ and we need this code to test it 

@giantswarm/team-clippy WDYT ?